src/globus_cli/termio/context.py

```diff
@@ -94,10 +95,12 @@ def env_interactive() -> bool | None:
 
 
 def term_is_interactive() -> bool:
-    try:
+    with suppress(ValueError):
         env = env_interactive()
         if env is not None:
             return env
-    except ValueError:
-        pass
```

Comment:
If you're alright doing so, would you revert this change? `env_interactive` uses distutils so we need to fix it -- and I plan to have it return `None` rather than raise. I'll do that right now, so leaving it unchanged here will avoid a conflict.

Reply:
Yep, I saw an approved the other PR and prefer the change there 👍